From abb19656fe813b9c54e9edf42e14d54b7b144394 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Fri, 10 Feb 2012 14:47:19 +0800
Subject: [PATCH 445/691] ENGR00174155-3 i.mx6sdl: enable i.mx6solo config by
 default

remove arm_freq=800 from default env thus keep 1GHz for kernel

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 include/configs/mx6dl_arm2.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/mx6dl_arm2.h b/include/configs/mx6dl_arm2.h
index 5c96b11..ed8974e 100644
--- a/include/configs/mx6dl_arm2.h
+++ b/include/configs/mx6dl_arm2.h
@@ -119,7 +119,7 @@
 		"kernel=uImage\0"				\
 		"nfsroot=/opt/eldk/arm\0"				\
 		"bootargs_base=setenv bootargs console=ttymxc0,115200 "\
-			"nosmp arm_freq=800 enable_wait_mode=off\0"     \
+			"nosmp enable_wait_mode=off\0"     \
 		"bootargs_nfs=setenv bootargs ${bootargs} root=/dev/nfs "\
 			"ip=dhcp nfsroot=${serverip}:${nfsroot},v3,tcp\0"\
 		"bootcmd_net=run bootargs_base bootargs_nfs; "		\
-- 
1.7.7.4

